docs/data-sources/app.md

```diff
@@ -0,0 +1,24 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "datadog_app Data Source - terraform-provider-datadog"
+subcategory: ""
+description: |-
+  Use this data source to retrieve information about an existing Datadog App from the App Builder product, for use in other resources.
```

Comment:
```suggestion
  Use this data source to retrieve information about an existing Datadog App from App Builder for use in other resources.
```

Reply:
An alternative suggestion to avoid using the word "Use" twice in the same sentence: "This data source retrieves information about an existing Datadog App from App Builder for use in other resources."